### ISSUE TITLE
Unable to train a loaded model

### ISSUE THREAD

**Issue by pmiller10** [IndicoDataSolutions/finetune#158]
**Describe the bug**
After I load a saved model, I'm unable to train it again. I've reproduced the bug on two different datasets.

    ValueError: Operation name: "NoOp_1"
    op: "NoOp"
     is not an element of this graph.

**Minimum Reproducible Example**
```
from finetune import Classifier
from sklearn.datasets import fetch_20newsgroups
    
dataset = fetch_20newsgroups()
trainX, trainY = dataset.data[:100], dataset.target[:100]
model = Classifier(n_epochs=1)
model.fit(trainX, trainY)
model.save('repro')
model = Classifier.load('repro')
model.fit(trainX, trainY)
```

**Comment by madisonmay** [comment]
Thanks for the bug report! Will look into this as soon as possible.  Could you give me the git hash of the commit you're on, or the version of the finetune package you're running?

```python
import finetune
print(finetune.__version__)
```

**Comment by pmiller10** [comment]
Sure, it's version `0.4.1`

**Comment by madisonmay** [comment]
Hi @pmiller10,

This should be fixed on our most recent branch (0.5.0). You can update your package either by pulling the development branch or by running:

```
pip install finetune --upgrade
```

You'll unfortunately have to retrain the model as I believe the serialization format has changed a bit between versions.

Hope that helps.

**Comment by pmiller10** [comment]
It looks good, thanks!
